Articles/Blog/2023-03-08-productivity-practices-and-tools.md

```diff
@@ -0,0 +1,155 @@
+# Busy != Productive
+
+#### Contributed by [Robert Underwood](https://robertu94.github.io) and [Mark C. Miller](https://github.com/markcmiller86)
+
+We've all heard the slogans...*"Do more with less"*, *"Work smarter not harder"*, *"Failing to plan means planning to fail"*, *"Slower is faster"*.
+
+Some critical skills *effective* software professionals need to master include managing a wide variety of tasks, accounting for how they spend their time, and continuously improving their own practices.
+For many, journaling, task management, and time tracking tools will help with these practices.
+
+However, as David Sparks and Mike Schmitz hosts of the [Focused podcast](https://www.relay.fm/focused) put it, "life is about more than cranking widgets".
+The goal of these tools is not to enable getting more done (they probably will), but to enable people to be better stewards of their time and resources.
+While it may be satisfying to write another unit test, release a new feature, or resolve a stubborn bug, there is always more we *could* be doing.
+Mastering the practices and tools described here will help make more time available for what we believe we *should* be doing, most.
+
+### Productivity Journaling
+
+Productivity journaling is different from general journaling (e.g. keeping a diary).
```

Comment:
```suggestion
Journaling for productivity is different what most people think of as keeping a diary.
```
I prefer the phrase `Journaling for productivity` to `productivity journaling`.  Productivity Journaling isn't some special kind of journaling which is wholly distinct from other kinds of journaling, but rather something which has some distinctive attributes when used for a purpose

I'm also not sure that all of the thinkers that I am interacting with in my writing would distinguish a diary from a journal.  I would agree that it's different than the "Dear diary, today I ..." motif that was popular in the 1800s and in childhood folk tales, but that's not how everyone uses that term even if that is the majority usage.